keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/resource/BlockDeviceConfig.kt

```diff
@@ -0,0 +1,195 @@
+package com.netflix.spinnaker.keel.ec2.resource
+
+import org.springframework.boot.context.properties.ConfigurationProperties
+import org.springframework.boot.context.properties.EnableConfigurationProperties
+import org.springframework.stereotype.Component
+
+
+@ConfigurationProperties(prefix = "keel.plugins.ec2.volumes.defaults")
+class VolumeDefaultConfiguration (
+  var volumeType : String = "gp2",
+  var deviceName : String = "/dev/sdb",
+  var deleteOnTermination : Boolean = true
+)
+
+data class BlockDevice(
+  val volumeType : String,
+  val deviceName : String,
+  val size : Int, // in GB
+  val deleteOnTermination : Boolean
+)
+
+
+/**
+ * Default config info for EBS block devices
+ *
+ * Note: this class only has config data for a subset of AWS instance types. In particularly, keel only knows
+ * about instance types where the clouddriver default is to mount a single EBS volume at "/dev/sdb"
+ */
+@Component
```

Comment:
I think you need to put components in the `com.netflix.spinnaker.config` package for them to be automatically picked up by Spring while scanning?